server/controllers/finance/reports/reportAccounts/index.js

```diff
@@ -30,42 +38,27 @@ function document(req, res, next) {
 
   const params = req.query;
 
-  const title = {
-    accountNumber : params.account_number,
-    accountLabel  : params.account_label,
-    source        : params.sourceLabel,
-    dateFrom      : params.dateFrom,
-    dateTo        : params.dateTo,
-  };
-
-  params.user = req.session.user;
+  if (!params.source) {
+    throw new BadRequest('A source ID `source` must be specified.', 'ERRORS.BAD_REQUEST');
+  }
 
   if (!params.account_id) {
     throw new BadRequest('Account ID missing', 'ERRORS.BAD_REQUEST');
   }
 
+  params.user = req.session.user;
+  params.sourceLabel = sourceMap[params.source].key;
+
   try {
     report = new ReportManager(TEMPLATE, req.session, params);
   } catch (e) {
     return next(e);
   }
 
-  const dateFrom = (params.dateFrom) ? new Date(params.dateFrom) : new Date();
-
-  return AccountsExtra.getOpeningBalanceForDate(params.account_id, dateFrom)
```

Comment:
This may be the result of a bad merge - the opening balance should be included here.